src/main/java/mnsky/task/Task.java

```diff
@@ -0,0 +1,62 @@
+package mnsky.task;
+
```

Comment:
Could you perhaps add a header Javadoc comment for this class? I also noticed some subclasses of this class have no header comments as well.